lib/fizzy/execute.cpp

```diff
@@ -613,8 +611,8 @@ execution_result execute(
     const auto& code = instance.module.codesec[code_idx];
     auto* const memory = instance.memory.get();
 
-    std::vector<uint64_t> locals = std::move(args);
-    locals.resize(locals.size() + code.local_count);
+    std::vector<uint64_t> locals(args.size() + code.local_count, 0);
```

Comment:
Is this zeroing it out?

Reply:
Yes. First we allocate and zero for args + locals. Then we overwrite args with proper values.